libs/openFrameworks/gl/ofShadow.h

```diff
@@ -221,6 +304,14 @@ class ofShadow {
 	void initShaders(ofGLProgrammableRenderer & renderer) const;
 	
 	mutable std::map<ofGLProgrammableRenderer*,std::shared_ptr<Shaders>> shaders;
+	
 };
 
-std::vector<std::weak_ptr<ofShadow::Data> > & ofShadowsData();
+//namespace of{
```

Comment:
Can probably remove this now.

Reply:
Nice catch, submitted another PR since this was already merged #8441